src/js/plyr.d.ts

```diff
@@ -555,6 +562,12 @@ declare namespace Plyr {
     src?: string;
   }
 
+  export interface Elements {
+    buttons: {
+      play?: HTMLButtonElement | HTMLButtonElement[];
+    };
```

Comment:
```suggestion
      airplay?: HTMLButtonElement | HTMLButtonElement[];
      captions?: HTMLButtonElement | HTMLButtonElement[];
      fastForward?: HTMLButtonElement | HTMLButtonElement[];
      fullscreen?: HTMLButtonElement | HTMLButtonElement[];
      mute?: HTMLButtonElement | HTMLButtonElement[];
      pause?: HTMLButtonElement | HTMLButtonElement[];
      pip?: HTMLButtonElement | HTMLButtonElement[];
      play?: HTMLButtonElement | HTMLButtonElement[];
      restart?: HTMLButtonElement | HTMLButtonElement[];
      rewind?: HTMLButtonElement | HTMLButtonElement[];
      settings?: HTMLButtonElement | HTMLButtonElement[];
    };
    captions: HTMLElement|null;
    container: HTMLElement|null;
    controls: HTMLElement|null;
    fullscreen: HTMLElement|null;
    wrapper: HTMLElement|null;
```

Could you also please add some of the remaining elements?

Reply:
Done, sorry for the delay